Functions/FirebaseFunctions/FUNSerializer.m

```diff
@@ -153,7 +153,10 @@ - (nullable id)decodeWrappedType:(NSDictionary *)wrapped error:(NSError **)error
     NSNumberFormatter *formatter = [[NSNumberFormatter alloc] init];
     NSNumber *n = [formatter numberFromString:value];
     if (n == nil) {
-      *error = FUNInvalidNumberError(value, wrapped);
+      // Only assign the `error` parameter if it was passed in.
+      if (error != NULL) {
```

Comment:
Wouldn't `nil` be more idiomatic than `NULL` in this case?

Reply:
`NULL` represents an "empty" pointer in C, in contrast to `nil` representing an empty Objective-C object. Since we are checking non-empty pointer to an error object `NULL` looks more appropriate literal to use here.

Reply:
+1 to what @maksymmalyhin said. The analyzer also complained about a missing `NULL` check.